sdk/core/ci.yml

```diff
@@ -56,3 +56,8 @@ extends:
     TestSetupSteps:
```

Comment:
Lets remove setting ServicesToTest parameter.

Reply:
![image](https://user-images.githubusercontent.com/48036328/180872364-7729a51c-6ad2-4906-93bc-ec4c5835600d.png)
Here is another place using the parameter. 
To avoid too many changes involved, I will remove the tests related step only.

Reply:
I'm find with us doing that for now but a big part of this work is to remove the need for the ServiceToTest parameter. Can you please file a follow-up issue to remove that?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/30433